src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/DeploymentTests.cs

```diff
@@ -38,6 +38,18 @@ public void TestNewDeploymentFromTemplateFile()
         }
 
         [Fact]
+        public void TestNewDeploymentFromNonexistentTemplateFile()
+        {
+            ResourcesController.NewInstance.RunPsTest("Test-NewDeploymentFromNonexistentTemplateFile");
```

Comment:
There are no json recordings for these tests, which is causing an on-demand run failure.  These should all be attributed as check-in tests.
```
[Train(Category.AcceptanceType, Category.CheckIn)]
```